cas-server-documentation/installation/LDAP-Authentication.md

```diff
@@ -10,12 +10,12 @@ LDAP integration is enabled by including the following dependency in the Maven W
 <dependency>
      <groupId>org.jasig.cas</groupId>
      <artifactId>cas-server-support-ldap</artifactId>
-     <version>${cas.version}</version>
+     <version>${project.version}</version>
```

Comment:
with $cas.version, it does not compile, saying that there is no such version.

what is supposed action now? I redo it and make another PR?